lib/node_modules/@stdlib/blas/base/ssyr2/README.md

```diff
@@ -193,21 +193,36 @@ console.log( A );
 ### Usage
 
 ```c
-TODO
+#include "stdlib/blas/base/ssyr2.h"
 ```
 
-#### TODO
+#### c_ssyr2( order, uplo, N, alpha, \*X, strideX, \*Y, strideY, \*A, LDA )
 
-TODO.
+Performs the symmetric rank 2 operation `A = α*x*y^T + α*y*x^T + A`.
 
 ```c
-TODO
+float A[] = { 1.0f, 0.0f, 0.0f, 2.0f, 1.0f, 0.0f, 3.0f, 2.0f, 1.0f };
+float x[] = { 1.0f, 2.0f, 3.0f };
+float y[] = { 1.0f, 2.0f, 3.0f };
+
+c_ssyr2( 102, 121, 3, 1.0f, x, 1, y, 1, A, 3 );
```

Comment:
We should be using the enums from shared. We should not use hardcoded values as those are subject to change.